Cargo.toml

```diff
@@ -14,57 +14,28 @@ license = "MIT"
 circle-ci = { repository = "amethyst/rlua", branch = "master" }
 
 [features]
-default = ["builtin-lua54"]
-# Builds the correct version of Lua 5.4 inside the crate. If you want to link a
-# specialized version of lua into your binary, you can disable this feature to
-# do that, but care must be taken.  `rlua` makes at least the following
-# assumptions about the linked lua library:
-#   * LUA_INTEGER is long long
-#   * LUA_NUMBER as double
-#   * LUA_EXTRASPACE is at least pointer sized and has at least pointer alignment.
-#   * LUAI_MAXSTACK is 1_000_000
-builtin-lua54 = ["cc"]
-# Builds the correct version of Lua 5.3 inside the crate. If you want to link a
-# specialized version of lua into your binary, you can disable this feature to
-# do that, but care must be taken.  `rlua` makes at least the following
-# assumptions about the linked lua library:
-#   * LUA_INTEGER is long long
-#   * LUA_NUMBER as double
-#   * LUA_EXTRASPACE is at least pointer sized and has at least pointer alignment.
-#   * LUAI_MAXSTACK is 1_000_000
-builtin-lua53 = ["cc"]
+default=[]
```

Comment:
There should still be a default, which should be the builtin Lua 5.4 as before.

Reply:
That's the problem the default features are always enabled. So you end up enabling multiple versions of Lua. Not sure if you can make the selection exclusive.

Reply:
You can disable default features (e.g. `cargo build --no-default-features --features builtin-lua53` or `default-features = false` in a dependent crate's Cargo.toml).  The exclusive part is enforced by `build.rs` (it counts the enabled features and errors if != 1).